edit/service.vue

```diff
@@ -151,7 +156,33 @@ export default {
 
     updateServicePorts(servicePorts) {
       this.$set(this.value.spec, 'ports', servicePorts);
-    }
+    },
+
+    targetPortsStrOrInt(targetPorts = []) {
+      const neu = clone(targetPorts);
+      const isNumeric = new RegExp('^\\d+$');
+
+      neu.forEach((port, idx) => {
+        if (port?.targetPort && isNumeric.test(port.targetPort)) {
+          try {
+            port.targetPort = parseInt(port.targetPort, 10);
+          } catch (err) {
+            console.warn(`Could not cast target port to int`, port, idx, err); // eslint-disable-line no-console
+          }
```

Comment:
This is unreachable.  

1) Pretty sure there's nothing that satisfies the regex that could fail to parse
2) parseInt doesn't throw exceptions on errors, it returns NaN.

Reply:
Removed the unneeded try...catch.